monai/transforms/intensity/array.py

```diff
@@ -120,19 +121,24 @@ def __init__(
             minv: minimum value of output data.
             maxv: maximum value of output data.
             factor: factor scale by ``v = v * (1 + factor)``.
+
+        Raises:
+            ValueError: if `minv` and `maxv` are None `factor` must be non None
         """
         self.minv = minv
         self.maxv = maxv
         self.factor = factor
 
-    def __call__(self, img):
+    def __call__(self, img: np.ndarray) -> np.ndarray:
         """
         Apply the transform to `img`.
         """
         if self.minv is not None and self.maxv is not None:
             return rescale_array(img, self.minv, self.maxv, img.dtype)
-        else:
+        elif self.factor is not None:
             return (img * (1 + self.factor)).astype(img.dtype)
+        else:
+            raise ValueError("if `minv` and `maxv` are None `factor` must be non None")
```

Comment:
could you please add "." to the end of error message, just be same as your other PRs.
Thanks.